src/apis/v1/answer/controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Response } from 'express';
+
+import * as service from './service';
+import fmt from 'utils/formatter';
+import RequestWithUser from 'utils/rest/request';
+import { AnswerDto, ParamsAnswerDto, UpdateAnswerDto } from './dto/AnswerDto';
+
+export const getAnswer = async (req: RequestWithUser, res: Response) => {
```

Comment:
if this function find all of the object, you should be add s in the end Answer like Answers